src/node.cc

```diff
@@ -3824,7 +3825,10 @@ static void StartNodeInstance(void* arg) {
 
     bool more;
     do {
+      v8::platform::PumpMessageLoop(default_platform, isolate);
       more = uv_run(env->event_loop(), UV_RUN_ONCE);
+      v8::platform::PumpMessageLoop(default_platform, isolate);
```

Comment:
Note to self: this one can be moved into the `if (more == false) {` branch below.